SCF-Session-5/searches/Search.java

```diff
@@ -0,0 +1,62 @@
+package searches;
+
+/**
+ * This class contains methods to perform linear and binary searches.
+ * @author Ishita_Mathur
+ *
+ */
+public class Search 
+{
+	int lowerLimit = 0;
+	/**
+	 * This method performs linear search on an input array.
+	 * @param inputArray
+	 * @param element, this is the element to be searched in the array. 
+	 * @return the index at which the element is found.
+	 */
+	public int linearSearch(int inputArray[], int element)
+    	{
+		int upperLimit;
+		if (inputArray.length == 0)
+			throw new AssertionError("Array is empty");
+		
+		upperLimit = inputArray.length - 1;
+        
+		if (upperLimit < lowerLimit)
+        		return -1;
```

Comment:
Please suggest any use case for this if condition.

Reply:
done